src/libraries/System.Net.Quic/ref/System.Net.Quic.cs

```diff
@@ -49,7 +52,12 @@ public sealed class QuicStream : System.IO.Stream
         public override void Write(byte[] buffer, int offset, int count) => throw null;
         public long StreamId => throw null;
         public void AbortRead() => throw null;
```

Comment:
I assume `AbortRead` corresponds to `STOP_SENDING`? If so, this should have an error code.